### ISSUE TITLE
feature request - improve escape count summary plots (low priority)

### ISSUE THREAD

**Issue by bblarsen-sci** [dms-vep/dms-vep-pipeline-3#154]
When analyzing escape, the pipeline generates summaries of barcode counts for the no-antibody and antibody selections that currently look like this:
<img width="979" alt="Screenshot 2024-10-10 at 5 07 05 PM" src="https://github.com/user-attachments/assets/0a05eaba-13ff-4c3b-94db-87f50007bfcd">

Where the red lines show where the frequency cutoffs are drawn. This is fine, but personally I don't find it very intuitive, and I think looking at the whole distribution would be more helpful. Something more like this?

![visualization-7](https://github.com/user-attachments/assets/1beb72d6-77f0-4898-8eae-fb9ffafb91f3)

Where the gray chart shows the barcode frequencies for the no antibody control, and the gray dotted lines shows the currently selected cutoff. I think it better shows whether the specified cutoffs are appropriate depending on the read distribution.

**Comment by jbloom** [comment]
Yes, I think this could be fine. The main issue is that these plots would be much bigger if there are a lot of samples. But maybe that is OK particularly if rather than color-coding them you faceted them with text titles so there would be multiple per row.

Do you want to try to implement this?

**Comment by bblarsen-sci** [comment]
Yep, I can submit a pull request once I figure it out

**Comment by bblarsen-sci** [comment]
How's something like this? Right now, I'm including the log transformed counts of barcodes with 0 reads, but clipping it at -8. I could remove them all too, but I think it shows some useful information?

![visualization](https://github.com/user-attachments/assets/cf76d579-68c6-49c1-853b-acfdebd3ce2a)

**Comment by jbloom** [comment]
I agree this looks good. Maybe rather than setting to -8, it would be better to set to something like 10-fold less than the smallest non-zero frequency.
